examples/drag-and-drop/script.js

```diff
@@ -0,0 +1,33 @@
+var flyd = require('flyd');
+var flatMap = require('flyd/module/flatmap');
+var takeUntil = require('flyd/module/takeuntil');
+
+document.addEventListener('DOMContentLoaded', function() {
+  var dragElm = document.getElementById('drag');
+
+  var mousedown  = flyd.stream();
+  var mousemove  = flyd.stream();
+  var mouseup    = flyd.stream();
+
+  dragElm.addEventListener('mousedown', mousedown);
+  document.addEventListener('mousemove', mousemove);
+  document.addEventListener('mouseup', mouseup);
+
+  var mousedrag = flatMap(function(md) {
+    var startX = md.offsetX, startY = md.offsetY;
+
+    return takeUntil(mousemove.map(function(mm) {
```

Comment:
The `map` method on streams is not supposed to be used. It's there because it is needed to implement the fantasy land specification. Instead of `stream.map(fn)` one should use `flyd.map(fn, stream)`.